doc/src/vtr/server_mode/index.rst

```diff
@@ -202,7 +202,17 @@ Draw selected critical path elements example
 
   .. note:: If status is not 1, the field ***DATA*** contains error string.
 
+.. _ipa_client_for_testing_vpr_server_mode_functionality:
 
+IPA Client for Testing VPR Server Mode Functionality
+====================================================
+
+To test VPR server functionality, you can use a special client application called **IPAClient**, which is available in a public Git repository:  
```

Comment:
I think you should make this first sentence a bit more end-user friendly.

"The interactive path analysis (IPA) client connects to VPR's server mode and allows interactive visualization of timing paths and their placement and routing. This client application is called **IPAClient** and can also be used to test VPR's server mode functionality. It is available in a public Git repository:"

Reply:
resolved, c0662ccce45319c20e9615f470b8eda60ff9b967